AllReadyApp/Web-App/AllReady/tsconfig.json

```diff
@@ -1,19 +1,19 @@
 {
-  "compileOnSave": true,
-  "compilerOptions": {
-    "module": "system",
-    "noEmitOnError": true,
-    "noImplicitAny": false,
-    "removeComments": false,
-    "sourceMap": true,
-    "target": "es5"
-  },
-  "include": [
-    "wwwroot/js/**/*.ts",
-    "wwwroot/ts/**/*.ts"
-  ],
-  "exclude": [
-    "node_modules",
-    "bin/output"
-  ]
+    "compileOnSave": true,
+    "compilerOptions": {
+        "module": "system",
+        "noEmitOnError": true,
+        "noImplicitAny": false,
+        "removeComments": false,
+        "sourceMap": true,
+        "target": "es5"
```

Comment:
Updating to ex2018 will be another task!

Reply:
Typo :) - I'm pretty sure you mean ES2018.  I'm definitely not up to date with JavaScript and would leave that to others to decide. 

There are so many things related to the client side that needs to discussed:
- possibly moving from _Knockout_ to _Vue_ 
- using _libman_ for package management